src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -21,11 +23,21 @@
 import javafx.scene.paint.Color;
 
 import java.util.Locale;
+import java.util.Optional;
 
 import static com.faforever.client.chat.ChatColorMode.CUSTOM;
+import static com.faforever.client.preferences.LanguageChannel.*;
 
 public class ChatPrefs {
 
+  @VisibleForTesting
+  public static final ImmutableMap<String, LanguageChannel> LOCALE_LANGUAGES_TO_CHANNELS = ImmutableMap.<String, LanguageChannel>builder()
+      .put("fr", FRENCH)
```

Comment:
Are there not constants for that Locale.FRENCH.getLanguage() or something...

Reply:
I don't think there are.

Reply:
There are for some types. I change String type to Locale.